TombEngine/Game/Lara/lara_one_gun.cpp

```diff
@@ -1459,7 +1467,7 @@ void HandleProjectile(ItemInfo& projectile, ItemInfo& emitter, const Vector3i& p
 	bool hasHit = false;
 	bool hasHitNotByEmitter = false;
 	bool isExplosive = type >= ProjectileType::Explosive;
-	bool isShatterable = type != ProjectileType::Harpoon;
+	bool isShatterable = true;
```

Comment:
This is incorrect. Why do you remove this condition? Harpoon should remain stuck into the room geometry until it shatters by itself after some interval.